reference/v8.json

```diff
@@ -68,7 +68,7 @@
         "function": "interpolated",
         "zoom-function": true,
         "property-function": false,
-        "doc": "Position of the light source relative to lit (extruded) geometries, in [r radial coordinate, a azimuthal angle, p polar angle] where r indicates the distance from the center of an object to its light, a indicates the position of the light relative to 0° (like a clock), and p indicates the height of the light (from 0°, directly above, to 180°, directly below).",
+        "doc": "Position of the light source relative to lit (extruded) geometries, in [r radial coordinate, a azimuthal angle, p polar angle] where r indicates the distance from the center of an object to its light, a indicates the position of the light relative to 0° (0° corresponds to 12:00 on a clock, and degrees proceed clockwise; 0° with `light.anchor: viewport` corresponds to the top of the viewport, or with `light.anchor: map` to due north), and p indicates the height of the light (from 0°, directly above, to 180°, directly below).",
```

Comment:
“12:00 on a clock” is no longer necessary, because “top of the viewport” and “due north” are descriptive enough.

Replace:

> with `light.anchor: viewport`

with:

> when `light.anchor` is set to `viewport`

The iOS and macOS SDKs have a script that translates the construct to something more appropriate for the language, so it’s best to use it when possible.